### PR TITLE
BeltFix

### PR DESCRIPTION
**Pull request by AveryThePanda**
Allows TacShield and TacKnives to be held in Military belts

### CODE REVIEW

**Comment by AveryThePanda** [comment]
!review

**Comment by ParadoxSpace** [comment]
It's an entire riot shield, even if it folds. Why.

File: code/game/objects/items/weapons/storage/belt.dm [hunk 1]
```diff
@@ -237,8 +237,9 @@
 		/obj/item/device/multitool/hacktool,
 		/obj/item/reagent_containers/hypospray/combat,
 		/obj/item/stack/telecrystal,
-		/obj/item/device/radio
-		)
+		/obj/item/device/radio,
+		/obj/item/shield/riot/tact,
+		/obj/item/material/knife/tacknife)
```
Comment (NonQueueingMatt) [inline code/game/objects/items/weapons/storage/belt.dm]: Put this bracket on the line below please.

File: html/changelogs/averythepanda-changelog.yml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+# Optional: Remove this file after generating master changelog.  Useful for PR changelogs that won't get used again.
+delete-after: True
+
+# Any changes you've made.  See valid prefix list above.
+# INDENT WITH TWO SPACES.  NOT TABS.  SPACES.
+# SCREW THIS UP AND IT WON'T WORK.
+# Also, all entries are changed into a single [] after a master changelog generation. Just remove the brackets when you add new entries.
+# Please surround your changes in  double quotes ("), as certain characters otherwise screws up compiling. The quotes will not show up in the changelog.
+changes: 
+  - rscadd: ""Allows Military Belts to hold Tactical Knives and Tactical Riot Shields""
```
Comment (Geevies) [inline html/changelogs/averythepanda-changelog.yml]:
```suggestion
  - rscadd: "Allows Military Belts to hold Tactical Knives and Tactical Riot Shields"
```

File: code/game/objects/items/weapons/storage/belt.dm (hunk 1 above)
Comment (AveryThePanda) [reply code/game/objects/items/weapons/storage/belt.dm]: I don't understand what you mean. Do you mean the )? I don't see a bracket.

**Comment by AveryThePanda** [comment]
> It's an entire riot shield, even if it folds. Why.

It fits in the security belt. So.

File: code/game/objects/items/weapons/storage/belt.dm (hunk 1 above)
Comment (Doxxmedearly) [reply code/game/objects/items/weapons/storage/belt.dm]: Yes, the )

File: html/changelogs/averythepanda-changelog.yml (hunk 2 above)
Comment (AveryThePanda) [reply html/changelogs/averythepanda-changelog.yml]: I've edited it.

File: code/game/objects/items/weapons/storage/belt.dm (hunk 1 above)
Comment (AveryThePanda) [reply code/game/objects/items/weapons/storage/belt.dm]: Moved! :)

**Pull request merged**